python/src/trezorlib/ui.py

```diff
@@ -71,6 +78,11 @@ def echo(*args: Any, **kwargs: Any) -> None:
 
 
 def prompt(*args: Any, **kwargs: Any) -> Any:
+    # Disallowing hidden input when it would cause issues
```

Comment:
instead of the `kwargs` shenanigans below:
```
def prompt(text: str, prompt_hidden: bool = False, **kwargs: Any)
```
we never use `*args` so we can instead fetch the positional `text` parameter directly, which is useful (see below)